### PR TITLE
CASMINST-5853: set timezone directly

### PR DESCRIPTION
**Pull request by heemstra**
# Description

`ntp-upgrade-config.sh` was removed in CSM 1.3. A reference to this script was not removed from `ncn-image-modification.sh`. This commit sets the timezone directly instead of trying to use a non-existent script.

# Checklist Before Merging

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: scripts/operations/node_management/ncn-image-modification.sh [hunk 1]
```diff
@@ -311,8 +311,22 @@ function set_timezone() {
     if [ -n "$TIMEZONE" ]; then
         for squash in "${SQUASH_PATHS[@]}"; do
             squashfs_root="$(dirname "$squash")"/squashfs-root
-            echo "TZ=$TIMEZONE" > "$squashfs_root"/etc/environment
-            sed -i "s#^timedatectl set-timezone UTC#timedatectl set-timezone $NEWTZ#" "$squashfs_root"/srv/cray/scripts/metal/ntp-upgrade-config.sh
+
+            pushd "$squashfs_root"
+            if ! test -f usr/share/zoneinfo/"$TIMEZONE"; then
+                echo "Timezone file /usr/share/zoneinfo/$TIMEZONE does not exist"
```
Comment (rustydb) [inline scripts/operations/node_management/ncn-image-modification.sh:317]:
```suggestion
                echo >&2 "Timezone file /usr/share/zoneinfo/$TIMEZONE does not exist"
```

**Review by rustydb — CHANGES_REQUESTED** [review]
Looks good, just should print to stderr on error.

**Review by rustydb — APPROVED** [review]

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**